src/modules/markets/selectors/tags.js

```diff
@@ -1,46 +1,49 @@
-import memoizerific from 'memoizerific';
+import { createSelector } from 'reselect';
+import store from 'src/store';
+import { selectSelectedTagsState } from 'src/select-state';
+import { selectFilteredMarkets } from '../../markets/selectors/markets-filtered';
```

Comment:
As I’ve been re-skinning, I’ve been updating all paths to utilize the path aliases as outlined w/in the `.babelrc` file.